include/vcpkg/base/messages.h

```diff
@@ -228,11 +239,8 @@ namespace vcpkg::msg
 #define DECLARE_MSG_ARG(NAME, EXAMPLE)                                                                                 \
     constexpr static struct NAME##_t                                                                                   \
     {                                                                                                                  \
-        constexpr static const char* name() { return #NAME; }                                                          \
-        constexpr static const char* comment()                                                                         \
-        {                                                                                                              \
-            return sizeof(EXAMPLE) > 1 ? "example of {" #NAME "} is '" EXAMPLE "'.\n" : "";                            \
-        }                                                                                                              \
+        constexpr static const char* name = #NAME;                                                                     \
```

Comment:
Should these be `StringLiteral`s as well?

Reply:
These ones can't be because they're fed into fmt bits. (I tried)